packages/ui/src/helpers/statusValueCheck.ts

```diff
@@ -0,0 +1,47 @@
+type StatusProps = 'idle' | 'doing' | 'review' | 'done';
+type StatusCase = {
+  statusText: string;
+  timeStatus: string;
+  dateOrPoints: string;
+  isDoing: boolean;
+};
+
+export const statusValueCheck = (status: StatusProps, points: number, pointsMax: number, dateProps: Date) => {
```

Comment:
🤔 There is probably no need to create a separate helpers directory for this logic, because:
1. The function will be used only in the ProjectCard component.
2. This way you need to define some types twice, which makes code less DRY.

I think it would be better to move this function back to the component file.